### PR TITLE
[WIP] Molecular surface(s) implementation 

### PR DESCRIPTION
**Pull request by SunTzunami**
There are three types of molecular surfaces:
-  Van der Waals
-  Solvent Accessible
-  Solvent Excluded

This program aims to implement all the surfaces, currently the first two i.e. Van der Waals  Surface and Solvent Accessible Surface are implemented. This program is based on the paper "[Generating Triangulated Macromolecular Surfaces by Euclidean Distance Transform](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008140)" by Dong Xu and Yang Zhang.

### CODE REVIEW

**Comment by skoudoro** [comment]
Is this PR still in WIP/ draft? Can we start to review it?

**Comment by SunTzunami** [comment]
> Is this PR still in WIP/ draft? Can we start to review it?

Hi @skoudoro ! It has 2 out of 3 surfaces implemented and we need to find good smoothing algorithms (we were discussing this in a meeting). 

TLDR: It's not ready to be reviewed yet.

**Comment by skoudoro** [comment]
What is the status of this @SunTzunami? 

Thanks in advance for the feedback

**Comment by SunTzunami** [comment]
Hi @skoudoro ! I apologize for the delayed response.
We had to work on fixing the smoothing the last we discussed this. I tried different smoothing algorithms (as discussed in the meeting) but none could smoothen it.  
I had also made some Pybullet tutorials and a few other features unfortunately, I don't have access to my local PC as of now and I will get back to this by September when I get the access to my local PC.
